### ISSUE TITLE
Custom Quiver Slot [for your use]

### ISSUE THREAD

**Issue by MinecraftGuy926** [Strubbelkopp/Quiver#11]
I made this custom quiver slot so that I could have a backpack and quiver at the same time.
Here's the data if you want to add it to an update:
https://www.dropbox.com/scl/fi/yw9x4n9aa360pbxhztdp9/quiver_trinket_data.zip?rlkey=k7fkg5bzpxfotrkimdenhwee8&st=oj8soqmv&dl=0
![qs1](https://github.com/user-attachments/assets/829eb527-43f2-4ee4-9f56-bb23cb7157f3)
![qs2](https://github.com/user-attachments/assets/27315b3f-e117-4817-ba19-50698071558d)

**Comment by Strubbelkopp** [comment]
Thanks! I'll try to see if I can make it a config option as I wouldn't want that to be on by default

**Comment by rocascent** [comment]
i need this♥️

**Comment by rocascent** [comment]
how about put it in legging slot? and hang it by legs when rendering

**Comment by Strubbelkopp** [comment]
> how about put it in legging slot? and hang it by legs when rendering

That is something I have planned. Unfortunately lost the original model file and the recreating the positioning has been kind of a pain in the ass. Will definitely do it one day though. Could then either be worn in the leggings or the belt slot.

**Comment by ACE1337x** [comment]
i just want to say, i think this is the best quiver mod currently - the active arrow count display is nice and the quiver is easy to use and functions well, the only thing stopping me from using it is that this mod doesn't use the trinket's Quiver slot

**Comment by MinecraftGuy926** [comment]
Not sure if you want the trinket slot as pictured above, but here it is as a fabric mod. 

[Quiver-Trinket-Slot.zip](https://github.com/user-attachments/files/17581655/Quiver-Trinket-Slot.zip)

You'll have to rename it "Quiver-Trinket-Slot.jar" because github only lets me send zip files. 
Once you've renamed it, just drop it in your mods folder and it should work.

**Comment by ACE1337x** [comment]
awesome! thank you so much! it does work as a jar in the mods folder, but it also _almost_ works as a datapack as a zip as well!
im currently using Accessories on Forge with the Curios and Trinkets compat, and as a datapack zip its functional but the name and the icon are messed up, works flawlessly as a mod/jar though!

**Comment by MinecraftGuy926** [comment]
Yeah, if you don't use it as a mod the icon won't work, I have no idea why that is. You could just make a generic trinket slot without an icon if you prefer it as a datapack!
